### PR TITLE
chore: polish cicd scaffold readmes

### PR DESCRIPTION
**Pull request by IvanJobs**
to finish https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/13838883

### CODE REVIEW

**Comment by IvanJobs** [comment]
close to make it draft.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/cicd/azdo/README.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 - Teams app projects are version controlled by Azure DevOps.
-- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
-- An Azure service princial with necessary permissions.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program). The M365 account credentials are required for steps of provision, publish and deployment for SPFx projects. Any extra interactive verification steps should be disabled for the M365 account, and please check details in sections below.
+- An Azure service princial with necessary permissions. The Azure service principal credentials are required for steps of provision and deploy for Azure based projects.
 
 ## Steps
 After the pre-cooked pipelines are scaffolded successfully, the following steps are expected to be performed:
-1. Add the scaffolded yml files into Azure DevOps by `git add`,`git commit` and `git push`.
+1. Commit and push your project source code to azure devops remote repository, including the CI/CD yml files.
```
Comment (zyxiaoyuer) [inline packages/fx-core/templates/plugins/resource/cicd/azdo/README.md:10]: Azure DevOps

File: packages/fx-core/templates/plugins/resource/cicd/azdo/README.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
 
 ## Prerequisites
 - Teams app projects are version controlled by Azure DevOps.
-- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
-- An Azure service princial with necessary permissions.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program). The M365 account credentials are required for steps of provision, publish and deployment for SPFx projects. Any extra interactive verification steps should be disabled for the M365 account, and please check details in sections below.
```
Comment (zyxiaoyuer) [inline packages/fx-core/templates/plugins/resource/cicd/azdo/README.md:5]: add a "(optional)" word at the begining?

File: packages/fx-core/templates/plugins/resource/cicd/azdo/README.md [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
+- An Azure service princial with necessary permissions. The Azure service principal credentials are required for steps of provision and deploy for Azure based projects.
```
Comment (zyxiaoyuer) [inline packages/fx-core/templates/plugins/resource/cicd/azdo/README.md:6]: add a "(optional)" word at the begining?

**Review by eriolchan — DISMISSED** [review]

File: packages/fx-core/templates/plugins/resource/cicd/azdo/README.md [hunk 4]
```diff
@@ -2,14 +2,15 @@
...
 
 ## Steps
 After the pre-cooked pipelines are scaffolded successfully, the following steps are expected to be performed:
-1. Add the scaffolded yml files into Azure DevOps by `git add`,`git commit` and `git push`.
+1. Commit and push your project source code to azure devops remote repository, including the CI/CD yml files.
 1. Create corresponding Azure DevOps pipelines by following [Create your first Azure DevOps Pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=java%2Ctfs-2018-2%2Cbrowser).
-1. Configure necessary Azure DevOps Pipeline variables for your pipelines.
+1. Configure necessary Azure DevOps Pipeline variables if your pipelines require credentials by checking into the yml files.
+1. Trigger your pipelines automatically, manually or do customization (Check the `trigger:` or `pr:` section in yml files to find the triggers). More about triggers in azure devops, refer to [Triggers in Azure pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops).
```
Comment (zyxiaoyuer) [inline packages/fx-core/templates/plugins/resource/cicd/azdo/README.md:13]: Azure DevOps

Comment (IvanJobs) [reply packages/fx-core/templates/plugins/resource/cicd/azdo/README.md:13]: good catch.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**